# Brain-cancer-gene-analysis


## Aim: To identify which genes have differential gene expression profiles in 4 different cancers compared to healthy tissue.

# Background Information on Data
Link: (https://www.kaggle.com/datasets/brunogrisci/brain-cancer-gene-expression-cumida)

This dataset has gene expression data from 78 cancer microarray datasets that have been classified into 5 different categories:
1. **Ependymoma** (A type of brain cancer that begins that the ependyma or cells that line the fluid-filled cavities of the brain)
2. **Glioblastoma** (A type of brain cancer that begins in astrocytes or the cells that support neurons)
3. **Medulloblastoma** (A type of brain cancer that occurs in the lower part of the brain, affecting the cerebellum)
4. **Pilyocytic Astrocytoma** (A types of brain cancer that begins in the central nervous system or CNS, which includes the brain and spinal cord)
5. **Healthy tissue**

# What is the Main Goal?
11 years ago marked the completion of the Human Genome Project (2003), one of the most intensive and innovative projects in the history of genome research. The outcomes of this project included the generation of new tools to analyzed huge amounts of genomic data and identify genes that are associated with different diseases.

Since then, many skilled scientists have been able to use these tools to identify many gene targets of different diseases that have been used in diagnosis and gene therapies. 

In this project, the goal is to find genes that have differential gene expression profiles compared to healthy tissue, which may indicate which gene are genetic markers or targets for gene therapy.

# What are Miccroarrays?
Microarrays are a type of high throughput gene expression profiling method used today to understand which genes are expressed at high or low levels in different samples. 

For a more in-depth step-by-step procedure of Microarraying, check out this video from Henrik's Lab:
(https://www.youtube.com/watch?v=NgRfc6atXQ8)

# What are Differential Gene Expression Profiles?
This is a high-throughput method of measuring the activity or expression of many genes at all by quantifying the transcription of these genes. If a gene is upregulated, usually this is represented by high transcription output (however, sometimes this isn't true since there are many other post-transcriptional factors that control the expression level of genes). 

With differential gene expression analysis, you can generate profiles where specific genes have similar expression patterns. We can further classify these profiles into different biochemical pathways or functions (e.g., cell division, metabolism, cell death/apoptosis). With this biochemical pathway analysis, we can understand how the upregulated genes involved in those pathways impact the pathways and thus, may result in cancerous symptoms, namely uncontrolled cell growth.

This can aid with finding potential treatments that are more targetted for the specific cancer profile based on the specific genes that are upregulated. These treatements can be better than the first-line treatment of chemotherapy, as there may be less side-effects and less burden on the patients.

Additionally, with understanding which genes are upregulated in specific cancer profiles, can help with prediction of risks in offspring and family members. This becomes important such that specific lifestyle habits can make certain sceptible individuals at less risk of getting diagnosed with cancer later in their life. One thing to remember is that "just because you have the genes that have been upregulated in specific cancers, **doesn't mean that you will get cancer**". Cancer is a complex disease that involves genetics and environment; this is why we try to understand further the cases of cancer that do occur to help with the future population.



